src/appProvider.js

```diff
@@ -11,13 +11,20 @@ import AppLocale from './languageProvider';
 import bodhiTheme from './config/theme';
 import graphClient from './network/graphClient';
 import { store, history } from './redux/store';
-
 import '../src/style/styles.less';
 
+const language = (navigator.languages && navigator.languages[0]) ||
+                     navigator.language ||
+                     navigator.userLanguage;
+let browserLang = 'mandarin';
+if (language.includes('en')) {
```

Comment:
Need ensure language starts with `en` or `en-`.  `en` can be in the middle of the string